CalculatorConsoleApp.AlainNshimirimana/Program.cs

```diff
@@ -0,0 +1,170 @@
+    Console.WriteLine("M. Multiplication");
+    Console.WriteLine("R. Square Root");
+    Console.WriteLine("P. Raise to a power");
+    Console.WriteLine("V. View History (not functional at the moment)");
+    Console.WriteLine("E. Exit");
+    Console.Write("> ");
+    Console.ResetColor();
+    selection = Console.ReadLine().Trim().ToUpper();
+    Console.WriteLine();
+
+    while (selection != "A" && selection != "S" &&  selection != "D" && selection != "M" && selection != "E" && selection != "R" && selection != "P")
+    {
+        Console.WriteLine("Invalid selection. Please enter A, S, D, M, or E to exit");
+        Console.ForegroundColor = ConsoleColor.Cyan;
+        Console.Write("> ");
+        Console.ResetColor();
+        selection = Console.ReadLine().ToUpper();
+    }
+
+    Console.Clear();
+
+    switch (selection)
+    {
+        case "A":
+            Addition();
+            break;
+        case "S":
+            Subtraction();
+            break;
+        case "D":
+            Division();
+            break;
+        case "M":
+            Multiplication();
+            break;
+        case "R":
+            SquareRoot();
+            break;
+        case "P":
+            Power();
+            break;
+        default:
+            break;
+    }
+}
+
+void Power()
+{
+    operationType = "Power";
+    Console.ForegroundColor = ConsoleColor.Cyan;
+    Console.WriteLine("POWER");
+    Console.ResetColor();
+    Console.WriteLine("--------");
+    Console.Write("Base: ");
+    num1 = double.Parse(Console.ReadLine());
+    Console.Write("Power: ");
+    num2 = double.Parse(Console.ReadLine());
+    result = Math.Pow(num1, num2);
+    Console.WriteLine($"{num1} ^ {num2} = {result}");
+    numGames++;
+    MainMenu();
+}
+
+void SquareRoot()
+{
+    operationType = "Square Root";
+    Console.ForegroundColor = ConsoleColor.Cyan;
+    Console.WriteLine("SQUARE ROOT");
+    Console.ResetColor();
+    Console.WriteLine("--------");
+    Console.Write("Enter a number: ");
+    num1 = double.Parse(Console.ReadLine());
+    result = Math.Sqrt(num1);
+    Console.WriteLine($"√{num1} = {result}");
+    numGames++;
+    MainMenu();
+}
+
+void Multiplication()
+{
+    operationType = "Multiplication";
+    Console.ForegroundColor = ConsoleColor.Cyan;
+    Console.WriteLine("MULTIPLICATION");
+    Console.ResetColor();
+    Console.WriteLine("--------");
+    Console.Write("First number: ");
+    num1 = double.Parse(Console.ReadLine());
+    Console.Write("Second number: ");
+    num2 = double.Parse(Console.ReadLine());
+    result = num1 * num2;
+    Console.WriteLine($"{num1} × {num2} = {result:0.00}");
+    numGames++;
+    MainMenu();
+}
+
+void Division()
+{
+    operationType = "Division";
+    Console.ForegroundColor = ConsoleColor.Cyan;
+    Console.WriteLine("DIVISION");
+    Console.ResetColor();
+    Console.WriteLine("--------");
+    Console.Write("First number: ");
+    num1 = double.Parse(Console.ReadLine());
```

Comment:
I'm getting this error when trying to input something other than a number. 
![image](https://user-images.githubusercontent.com/108711476/210284048-e4a26eed-54db-4c5e-a7b7-f3477e6a2206.png)

Reply:
Fixed or at least made it so the app checks that the user entered a number